src/components/radio/RadioList.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { action } from '@storybook/addon-actions';
+
+import { Spacer } from '@chakra-ui/react';
+
+import RadioList from './RadioList.tsx';
+import { Box, Flex } from '../index';
+
+<Meta
+  title="Components/Forms/RadioList"
+  component={RadioList}
+  parameters={{
+    layout: 'fullscreen',
+  }}
+/>
+
+export const Container = ({ children }) => {
+  return (
+    <Box padding="xl" background="purple">
+      {children}
+    </Box>
+  );
+};
+
+export const DummyOption = (name) => {
+  return (
+    <Flex key={name}>
+      <div>
+        <div>A5 Coupé 2.0 40 TDO Sport qu. S-Tronic</div>
+        <div>2018 - 2020</div>
+      </div>
+      <Spacer />
```

Comment:
Do you think will you need to extract `Spacer` when you work on the table? Does it make sense to extracted it?

Reply:
It's just demo content I don't think we will end up using Grid since the content will not be evenly spaced